application/models/Survey.php

```diff
@@ -1256,4 +1257,31 @@ public function getsSurveyUrl()
         return $this->sSurveyUrl;
     }
 
+    /**
+     * @return Question[]
+     */
+    public function getQuotableQuestions()
+    {
+        $criteria=new CDbCriteria;
+        $criteria->select = 't.*';
+        $criteria->with=array(
+            'survey.groups',
+        );
+
+        $criteria->order ='`groups`.`group_order`, `t`.`question_order`';
```

Comment:
Thenks to you ;) . Looking at code : seems to be the best way : http://www.yiiframework.com/doc/api/1.1/CDbSchema#quoteColumnName-detail